openwisp_users/migrations/__init__.py

```diff
@@ -17,8 +20,33 @@ def set_default_organization_uuid(apps, schema_editor):
         )
         default_organization.full_clean()
         default_organization.save()
-    
-    # settings._OPENWISP_DEFAULT_ORG_UUID is used in 
-    # openwisp-radius.migrations, it helps to enable 
+
+    # settings._OPENWISP_DEFAULT_ORG_UUID is used in
+    # openwisp-radius.migrations, it helps to enable
     # users to migrate from freeradius 3
     settings._OPENWISP_DEFAULT_ORG_UUID = default_organization.pk
+
+
+def create_default_groups(apps, schema_editor):
+    group = apps.get_model('openwisp_users', 'group')
+
+    # To populate all the permissions
+    for app_config in apps.get_app_configs():
+        app_config.models_module = True
+        create_permissions(app_config, apps=apps, verbosity=0)
+        app_config.models_module = None
+
+    operator = group.objects.filter(name='operator')
+    if operator.count() == 0:
+        operator = group.objects.create(name='operator')
+        operator.permissions.set([])
+
+    user_manager = group.objects.filter(name='user manager')
+    if user_manager.count() == 0:
+        user_manager = group.objects.create(name='user manager')
+        user_manager.permissions.set([
+            Permission.objects.get(codename='add_user').pk,
+            Permission.objects.get(codename='change_user').pk,
+            Permission.objects.get(codename='view_user').pk,
```

Comment:
this permission is available only from django 2.1 onwards (finally!). In order to support older versions of django the simples thing you can do in this case is to build the list of permissions (except the view one) before the `set` call, then append the `view_user` permission within a `try, except DoesNotExist:` block.